packages/react-native-macos-init/README.md

```diff
@@ -0,0 +1,24 @@
+# react-native-macos-init
+
+CLI to bootstrap the addition of the macOS platform to `react-native` projects.
+
+## Usage
+
+Run this from an existing `react-native` project to install `react-native-macos` and generate initial project files for macOS.
```

Comment:
Should this doc make note of the RN version to use? I.e. that it only supports apps that use RN v0.60.0 atm?

Reply:
The script will attempt to pick the latest version of react-native-macos whose version number matches the version of react-native currently installed in the init'd project.   Same as in react-native-windows-init.   Does it need to enumerate versions in the readme?  (they'll always be changing)